diesel/src/expression/array_comparison.rs

```diff
@@ -53,12 +53,14 @@ where
 
 impl<T, U> NonAggregate for In<T, U>
```

Comment:
Why don't we derive this two impls too?

Reply:
Because we don't want `U: NonAggregate` as a bound.